src/containers/search_result.jsx

```diff
@@ -158,53 +189,6 @@ class SearchResult extends Component {
     );
   }
 
-  renderSortBy() {
-    const { search: { query } } = this.props;
-
-    const Container = styled.div`
-      display: flex;
-      align-items: center;
-    `;
-
-    const options = [
-      {
-        value: SEARCH_SORT_BY.RELEVANCE,
```

Comment:
If this function is useless, the relevant code should also be cleaned. I mean the related constants defined in `constants/index.js`, such as **SEARCH_SORT_BY**.

Reply:
finished